openmc/geometry.py

```diff
@@ -281,9 +283,9 @@ def get_materials_by_name(self, name, case_sensitive=False, matching=False):
             The name to match
         case_sensitive : bool
             Whether to distinguish upper and lower case letters in each
-            material's name (default is True)
+            material's name (default is False)
```

Comment:
Oh, no, I just happened to notice this along the way. That would be sweet if pylint were that smart, but alas, it's not...